cunumeric/module.py

```diff
@@ -388,7 +413,14 @@ def full_like(a, value, dtype=None):
 # From existing data
 
 
-def array(obj, dtype=None, copy=True, order="K", subok=False, ndmin=0):
+def array(
+    obj: Any,
+    dtype: Optional[npt.DTypeLike] = None,
+    copy: bool = True,
+    order: str = "K",
```

Comment:
I want to go back and define better literal union types for this and others, I just didn't want to interrupt to flow of finishing this large module, since it will take a little research

Reply:
Also private :/ 

https://github.com/numpy/numpy/blob/main/numpy/__init__.pyi#L969-L971